src/tests/pool.spec.ts

```diff
@@ -1,18 +1,9 @@
-import {
-  ORAI,
-  buildMultipleExecuteMessages
-} from '@oraichain/oraidex-common';
+import { ORAI, buildMultipleExecuteMessages } from '@oraichain/oraidex-common';
```

Comment:
seems like we have very little tests to cover this big refactor. Any chance we can cover at least 40% of public behaviors?